vscode-extension/src/welcomePage.html

```diff
@@ -2,7 +2,123 @@
 <html lang="en">
 
 <body>
-    <h1>Welcome to my extension!</h1>
-</body>
+    <h1>Welcome to AIConfig Editor!</h1>
+
+    <!-- <p>
+        How do you do, fellow kids? My name is Screve Busheggy 
+        (not to be confused with famous actor Steve Buscemi), 
+        and I'm here to get you started with the AIConfig Editor extension 
+    </p>
+    <img src="https://s3.amazonaws.com/lastmileai.aiconfig.public/how_do_you_do_fellow_kids.jpeg" alt="Showing kids how cool AIConfig Editor is" width="250" height="200"> -->
+
+    AIConfig Editor is a tool that allows you to <b>create, edit, and manage AIConfig files</b>.<br>
+    <b> Tip:</b> To learn more about AIConfigs and why they're useful, check out the FAQ 
+    section at the bottom of this page.
+    <br>
+    <p>
+        <img src="https://s3.amazonaws.com/lastmileai.aiconfig.public/Drake-Hotline-Bling.jpg" alt="Drake AIConfig meme" width="250" height="250">
+    </p>
+
+    Get started by trying out some of these commands! Press ("CMD" + "SHIFT" + P) 
+    to open the command palette and type in "AIConfig":
+
+    <ul>
+        <li>"AIConfig: Create New" -> Creates a new file (in JSON or YAML format)</li>
+        <li>"AIConfig: Share (get permalink)" -> Creates a link to read-only state of your AIConfig file. Use this to share with others :D</li>
+        <li>"AIConfig: Welcome" -> Opens this window that you can refer to again!</li>
+    </ul>
+
+    You can also open existing files that end in ".aiconfig.json" or ".aiconfig.yaml" 
+    (note that there are two periods in the filename!). <br>
+    Example: "my_cool_file.aiconfig.json"
+
+
+
+
+    <!-- <li>AIConfig: Initialize Extension -> Creates a link to read-only state of your AIConfig file</li>
 
+
+
+    <p></p>Here's what the raw data looks like:
+
+    Pretty ugly amiright? This is why we have the AIConfig Editor. It lets you play around with 
+    generative AI models using AIConfig files with easily button clicks. Think of it like being able
+    to open up a Word document with Microsoft Word instead of having to write it in Notepad. -->
+
+
+    <h1> FAQ </h1>
+    <p>
+        <b>Q: What is AIConfig?</b> <br>
+
+        A: AIConfig files are files in JSON or YAML format where you can define the 
+        structure of generative AI workflows. Each AIConfig file contains a set 
```

Comment:
I think we should speak more to configuring prompt inputs, models and settings instead of calling out structure of workflows

Reply:
Added an image from Gradio workbooks showing general instructions